src/entity/Organization.ts

```diff
@@ -61,6 +122,33 @@ export class Organization {
   @Column({ type: 'jsonb', default: {} })
   subscriptionFlags: z.infer<typeof organizationSubscriptionFlagsSchema>;
 
+  @Column({ type: 'jsonb', default: '[]' })
+  links: z.infer<typeof organizationLinksSchema>;
+
+  @Column({ type: 'text', default: null })
+  website: string;
+
+  @Column({ type: 'text', default: null })
+  description: string;
+
+  @Column({ type: 'text', array: true, default: null })
+  perks: string;
+
+  @Column({ type: 'numeric', default: null })
+  founded: number;
+
+  @Column({ type: 'text', default: null })
+  location: string;
+
+  @Column({ type: 'text', default: null })
+  size: CompanySize;
+
+  @Column({ type: 'text', default: null })
+  category: CompanyCategory;
```

Comment:
I think string will work better here, else we probably end up runing tons of changes over and over.